export_doc.py

```diff
@@ -1,3 +1,6 @@
+"""
+Export doc
+"""
```

Comment:
Hardly usefull

Reply:
Indeed. But it at least silences pylint.

Reply:
Yes, but that's not a good reason to commit